src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentationOptions.cs

```diff
@@ -53,5 +53,10 @@ public class AspNetCoreInstrumentationOptions
         /// The type of this object depends on the event, which is given by the above parameter.</para>
         /// </remarks>
         public Action<Activity, string, object> Enrich { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the exception will be recorded or not.
+        /// </summary>
+        public bool RecordException { get; set; }
```

Comment:
Any thoughts on making this false by default? 
My suggestion is make this opt-in. Storing exception is somewhat expensive, so lets do this only if users opt-in

Reply:
since we are not setting anything, it is false by default.